stable/elastalert/templates/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+{{- $root := . }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "elastalert.fullname" . }}
+spec:
+  replicas: 1
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
+  template:
+    metadata:
+      name: {{ template "elastalert.fullname" . }}-elastalert
+      labels:
```

Comment:
Only `app` and `release` labels should go here. I don't think we need a `component` label since there is no multiple components in this chart. All four labels should go below `metadata`. The configmaps have it correctly.